### PR TITLE
Bump AMF version

### PR DESCRIPTION
**Pull request by emilianoascona**

### CODE REVIEW

File: dependencies.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 version=6.1.0-SNAPSHOT
-amf=5.4.7
+amf=5.4.8-RC.1
 amf.custom-validator.js=1.6.0
```
Comment (mansillafederico) [inline dependencies.properties:3]: I think we need use `1.7.0` as `amf.custom-validator-scalajs=0.5.8-RC.0`

**Review by mansillafederico — DISMISSED** [review]
LGTM, just think we need one more bump

File: dependencies.properties (hunk 1 above)
Comment (emilianoascona) [reply dependencies.properties]: You're right!

**Review by mansillafederico — APPROVED** [review]
LGTM!

**Pull request merged**